packages/runtimes/hmr/src/HMRRuntime.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+import type {BundleGroupNode} from '@parcel/types';
+
+import {Runtime} from '@parcel/plugin';
+import {readFile} from '@parcel/fs';
+
+const HMR_RUNTIME = './loaders/hmr-runtime';
+
+let hmrRuntimeCode = null;
+export default new Runtime({
```

Comment:
Unrelated: this is pretty close to the JSRuntime 😄  I wonder if all runtime will end up looking like this. Maybe the runtime api could just wrap this pattern 🤔

cc @devongovett

Reply:
What is the generalizable pattern? Some logic, then conditionally call addRuntimeAsset?